setup.cfg

```diff
@@ -0,0 +1,3 @@
+[tool:pytest]
+minversion = 2.8
```

Comment:
afaik the format ``[tool:pytest]`` need pytest 3.0, otherwise nasty things happen, e.g.: https://github.com/astropy/astropy/issues/5695